<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>789</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TouristManager 1.0</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="AddPackage">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>160</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Trip</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RegisterTourist">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>160</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Tourist</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FindPakcage">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>160</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Find Trip</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FindTourist">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>160</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Find Tourist</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>789</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
